tests/conftest.py

```diff
@@ -20,9 +20,43 @@
 import pytest
 import pytest_asyncio
 import websockets
+from http_server.test_web_server import StaticWebServer
 from test_helpers import execute_command, get_tree, goto_url, read_JSON_message
 
 
+@pytest_asyncio.fixture(scope='session')
+def static_web_server():
+    # TODO: provide a path to a certificate for HTTPS once `ignoreHTTPSErrors`
+    # capability is implemented.
+    server = StaticWebServer()
+    yield server
+    server.shutdown()
+
+
+@pytest_asyncio.fixture
+def url(static_web_server):
+    def url(code=200, content=b'', headers=None):
+        return static_web_server.url(code, content, headers)
+
+    return url
+
+
+@pytest_asyncio.fixture
+def url_200(url):
+    def url_200(content='<html><body>default 200 page</body></html>'):
+        return url(200, bytes(content, 'utf-8'))
+
+    return url_200
+
+
+@pytest_asyncio.fixture
+def url_301(url, url_200):
```

Comment:
nit: let's call it `url_permanent_redirect` (since it's not only sets 301 but also the location header) and perhaps remove the `url_200` in favour of url with default params?

Reply:
I'm fine with Alex's suggestion.